mirrord/cli/src/config.rs

```diff
@@ -217,3 +221,37 @@ pub(super) enum OperatorCommand {
         namespace: OperatorNamespace,
     },
 }
+
+#[derive(ValueEnum, Clone, Default, Debug)]
+pub enum Format {
+    #[default]
+    Json,
+}
+
+impl FromStr for Format {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "json" => Ok(Format::Json),
+            _ => Err(format!("invalid format: {}", s)),
+        }
+    }
+}
+
+#[derive(Args, Debug)]
+pub(super) struct LsArgs {
+    /// Specify the format of the output.
+    #[arg(
+        short = 'o',
+        long = "output",
+        value_name = "FORMAT",
+        value_enum,
+        default_value = "json"
```

Comment:
https://stackoverflow.com/questions/73184371/provide-default-value-for-argenum-in-rust-clap-cli

Reply:
what's is `value_name`? couldn't find it

Reply:
value_name is just changes what is printed in the arg, when -h is used
```
  -o, --output <FORMAT>  
  ```